clients/admin-ui/src/features/user-management/UserForm.tsx

```diff
@@ -47,6 +57,7 @@ export interface Props {
 }
 
 const UserForm = ({
+  user,
```

Comment:
hmm why do we want the `user` here?

Reply:
we need `user` in `PasswordManagement` -> `DeleteUserModal`. If we move the `DeleteUserModal` out of `PasswordManagement`, we'll still need a `user` passed in here, I believe